.gitignore

```diff
@@ -20,3 +20,8 @@
 *.sublime-project
 *.sublime-workspace
 *~
+
+deploy-recorder.lst
+/hack/go
+/hack/pkg
+/hack/*.tar.gz
```

Comment:
```suggestion
/hack/*.tar.gz

```

Reply:
To ensure the version of go that is used, during the installation of the dependencies, I download the version that is required and then remove it. 
This line ensures that if an error occurs, the go binary is not pushed.

Reply:
I'm just adding the newline.. :)